hack/install.sh

```diff
@@ -238,6 +238,8 @@ do_install() {
 			;;
 
 		fedora|centos)
+			# The dist_version from lsb_release is wrong, so grab it again to be safe.
+			dist_version="$(rpm -q --whatprovides redhat-release --queryformat "%{VERSION}\n")"
```

Comment:
Nope. The other PR fixes the case when lsb_release *isn't* installed. This fixes the case when it is, because the logic to determine distro and version is skewed to Debian+derivatives. 

As I said in the PR, a possibly more elegant fix is to remove the dist_version check completely from the non-Debian distros and only check it later.